packages/components/src/components/hds/table/index.hbs

```diff
@@ -13,7 +13,9 @@
   <thead class="hds-table__thead">
     {{#if @columns}}
       <Hds::Table::Tr
+        @sortOrder={{if (eq this.sortBy @selectableColumnKey) this.sortOrder}}
```

Comment:
It is the `sortOrder` of the table component.

It's the same way we pass the value to the `ThSort` component like

```
<Hds::Table::ThSort
              @sortOrder={{if (eq column.key this.sortBy) this.sortOrder}}
              @onClickSort={{fn this.setSortBy column.key}}
              @align={{column.align}}
              @width={{column.width}}
              @tooltip={{column.tooltip}}
            >```


It will either be `undefined`, `asc`, or `desc`. When it's `undefined`, that column is unsorted, and when it's defined, it's sorted by the specified `sortOrder`

Reply:
Nevermind, I see what you mean. sortOrder is misleading because it will be undefined when sorting by another value, which works but is confusing and incorrect